src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/AvoidMultipleConcatStatements.xml

```diff
@@ -36,7 +36,7 @@ public class AvoidMultipleConcatStatementsTest {
 
 		logStatement = logStatement + values.get(0);
 		logStatement = logStatement + values.get(1); // bad
-        logStatement = logStatement + values.get(2); // bad
+        logStatement = logStatement + values.get(2); // not reported to prevent violation overload
```

Comment:
so if the case of `get(1)` gets fixed, the second one is reported in new PMD run?